pages/agent/gcloud.md.erb

```diff
@@ -0,0 +1,183 @@
+
+Open [Google Cloud Shell](https://cloud.google.com/shell/) or your own console with [gcloud](https://cloud.google.com/sdk/gcloud/) installed and authenticated. You'll need to configure kubectl to talk to your new cluster. The console includes a "Connect" button which shows the exact command to run:
+
+<%= image "connect-to-cluster.png", size: "2878x1531", alt: "Screenshot of cluster connection settings in the Google Cloud Console" %>
+
+```
+$ gcloud container clusters get-credentials INSERT-YOUR-CLUSTER-NAME \
+    --zone INSERT-YOUR-ZONE --project INSERT-YOUR-PROJECT
+Fetching cluster endpoint and auth data.
+kubeconfig entry generated for YOUR-CLUSTER-NAME
+```
+
+Double check it's all working:
+
+```
+$ kubectl version
+Client Version: version.Info{Major:"1", Minor:"5", GitVersion:"v1.5.2", GitCommit:"08e099554f3c31f6e6f07b448ab3ed78d0520507", GitTreeState:"clean", BuildDate:"2017-01-12T04:57:25Z", 
+GoVersion:"go1.7.4", Compiler:"gc", Platform:"linux/amd64"}
+Server Version: version.Info{Major:"1", Minor:"5", GitVersion:"v1.5.2", GitCommit:"08e099554f3c31f6e6f07b448ab3ed78d0520507", GitTreeState:"clean", BuildDate:"2017-01-12T04:52:34Z", 
+GoVersion:"go1.7.4", Compiler:"gc", Platform:"linux/amd64"}
+```
+
+Create a [secret](https://kubernetes.io/docs/user-guide/secrets/#using-secrets-as-environment-variables) with your agent registration token:
+
+```shell
+$ kubectl create secret generic buildkite-agent --from-literal token=INSERT-YOUR-AGENT-TOKEN-HERE
+secret "buildkite-agent" created
+```
+
+Create a [deployment](https://kubernetes.io/docs/user-guide/deployments/#creating-a-deployment) to start an agent:
+
+```shell
+$ cat | kubectl apply -f -
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+ name: buildkite-agent
+spec:
+ replicas: 1
+ template:
+   metadata:
+     labels:
+       app: buildkite-agent
+   spec:
+     containers:
+       - name: buildkite-agent
+         image: buildkite/agent
+         imagePullPolicy: Always
+         securityContext:
+           privileged: true
+         env:
+           - name: BUILDKITE_AGENT_TOKEN
+             valueFrom: {secretKeyRef: {name: buildkite-agent, key: token}}
+         volumeMounts:
+           - name: docker-binary
+             mountPath: /usr/bin/docker
+           - name: docker-socket
+             mountPath: /var/run/docker.sock
+     volumes:
+       - name: docker-binary
+         hostPath: {path: /usr/bin/docker}
+       - name: docker-socket
+         hostPath: {path: /var/run/docker.sock}
+# (press Control-D)
+deployment "buildkite-agent" created
+```
+
+Now you should have a [pod](https://kubernetes.io/docs/user-guide/pods/) running:
+
+```
+$ kubectl get pods
+NAME                               READY     STATUS    RESTARTS   AGE
+buildkite-agent-3220167863-dtzcf   1/1       Running   0          30s
+```
+
+And you can see the logs:
```

Comment:
It's a good way to verify that the agent is actually running and connected, and folks often end up asking us how to see the logs. It's something we include on most of the other installation guides.

Should I hint at this here?